src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private static final String[] COLORS = {"Red", "Green", "Blue", "Yellow", "Black"};
```

Comment:
why don't you use an Enum for these color values?

Reply:
please use existing enum and its .name() method in order to get string representation